config/locales/en.yml

```
@@ -102,15 +102,24 @@ en:
         title: Local authority details
         rows:
           local_authority: Local authority
+  note:
+    index:
+      add_note_button: Add note
+      no_notes_yet: No notes yet
+    create:
+      success: Note has been created successfully
```

```suggestion
      success: Note has been added successfully
```

Maybe "added" is a better verb, to match what a user is doing conceptually?